config.yml

```diff
@@ -37,6 +37,9 @@ redis_activity_consumers: 8
 # Number of concurrent connections to Redis in the connection pool.
 redis_connection_pool_size: 20
 
+# Max number of seconds BLPOP waits before returning anyway. Must be a positive integer. 0 means wait indefinitely.
+redis_blpop_timeout: 10
```

Comment:
I closed this pr, new pr opened with this value with default at 0.

https://github.com/Imgur/incus/pull/73